io.github.chilledheart.yass.appdata.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.chilledheart.yass</id>
+  <name>YASS</name>
+  <developer id="letshack.info">
+    <name>Chilledheart</name>
+  </developer>
+  <developer_name>Chilledheart</developer_name>
+  <summary>Lightweight and efficient, socks/http forward proxy</summary>
+  <description>
+    <p>Free and open source software for socks/http forward proxy including http2/naiveproxy/https/socks4/socks5/shadowsock protocols</p>
+  </description>
```

Comment:
Can you add some more description?

Reply:
Done in 2fbfe8192925777bace6abb083e02809bbca82fb.